internal/server/dashboard_test.go

```
@@ -0,0 +1,96 @@
+	return mux
+}
+
+func TestDashboard_ServesRootWithMarker(t *testing.T) {
+	srv := httptest.NewServer(buildDashboardTestMux())
+	defer srv.Close()
+
+	resp, err := http.Get(srv.URL + "/")
+	if err != nil {
+		t.Fatalf("GET /: %v", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)